src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ClearNuGetLocalResourcesWindow.xaml

```diff
@@ -28,6 +29,11 @@
       </ResourceDictionary.MergedDictionaries>
     </ResourceDictionary>
   </Window.Resources>
+  <Window.CommandBindings>
+    <CommandBinding
+    Command="{x:Static nuget:PackageManagerControlCommands.OpenExternalLink}"
+    Executed="ExecuteOpenExternalLink" />
```

Comment:
These two lines should be indented once to not line up with their parent element.